core/chain-configs/src/genesis_config.rs

```diff
@@ -99,6 +103,10 @@ pub struct GenesisConfig {
     /// Fishermen stake threshold.
     #[serde(with = "u128_dec_format")]
     pub fishermen_threshold: Balance,
+    /// The minimum stake required for staking is last seat price divided by this number.
+    #[serde(default = "default_minimum_stake_divisor")]
+    #[default(10)]
+    pub minimum_stake_divisor: u64,
```

Comment:
How about using Rational instead of encoding the `1/divisor` implicitly?

Reply:
I don't think that's needed (at least for now) because the stake is usually a large number (due to yotcoNEAR) and for example there would very little difference if we divide by 33 or multiple by 3/100.